npc/custom/jobmaster.txt

```diff
@@ -504,6 +526,7 @@ OnInit:
 	setarray .Req_Fourth[0],200,70;		// Minimum base level, job level to change to fourth class
 	setarray .Req_Exp_NJ_GS[0],99,70; 	// Minimum base level, job level to turn into Expanded Ninja and Gunslinger
 	setarray .Req_Exp_SNOVI[0],99,99; 	// Minimum base level, job level to turn into Expanded Super Novice
+	setarray .Req_SHandler[0],200,60;	// Minimum base level, job level to turn into Spirit Handler
```

Comment:
Not sure if these are the correct levels

Reply:
https://www.youtube.com/watch?v=kuCVZv21Amc

The person in this video is level 200/60.

Reply:
200/60 on https://www.divine-pride.net/tools/skilltree/4308